### ISSUE TITLE
Ability to Temporarily Lock Folders in Workspace

### ISSUE THREAD

**Issue by Donchuks02** [microsoft/vscode#232941]
It would be helpful to have a feature that allows users to temporarily lock project folders they aren’t actively working on within a workspace. The locked folders could display a pop-up notification each time a user attempts to access them, preventing accidental edits and helping to maintain focus on the active folder. This feature would be particularly useful for developers working in large workspaces with multiple projects.

**Comment by albertosantini** [comment]
I think you know `files.exclude` setting.

Hope that helps.

**Comment by Donchuks02** [comment]
> I think you know `files.exclude` setting.
> 
> Hope that helps.

Yes, but for easy access, I'd suggest a lock option should be included in the options list for a folder.

**Comment by gjsjohnmurray** [comment]
Perhaps a candidate for an extension that manipulates the `files.readonlyInclude` setting.

**Comment by bpasero** [comment]
This can be done via `files.readonlyInclude` setting, but is not surfaced as an option in the explorer menu yet.
